Facebook login

jizavala opened the pull request:
- Modifica el gemfile para incluir las gemas devise, onmiauth-facebook, wdm y certified. 
- Agregado el controllador para el callback del onmiauth. 
- Modificado el routes para implementar los controllers del onmiauth. 
- Agregado el modelo Users. 
- Agregada la vista de la pagina que contendra el login mediante facebook y su respectivo logout.

jizavala committed:
Modifica el gemfile para incluir las gemas devise, onmiauth-facebook, wdm y certified. Agregado el controllador para el callback del onmiauth. Modificado el routes para implementar los controllers del onmiauth. agregado el modelo Users. Agregada la vista de la pagina que contendra el login mediante facebook y su respectivo logout

jizavala committed:
Agregado bootstrap (NO GEM)
Agregada galeria de imagenes para la pagina principal que se muestra al loguearse correctamente con facebook.

escoberik commented on Gemfile:
```diff
@@ -35,6 +41,12 @@ gem 'jbuilder', '~> 2.5'
 # gem 'capistrano-rails', group: :development
 
+#Use wdm (Windows Directory Monitor)
+gem 'wdm'
```
Esta gema agrégala solo en `group: :development`

jizavala replied:
Agregada gem wdm al group development.

escoberik commented on Gemfile:
```diff
@@ -8,6 +8,12 @@ end
+#Use omniauth-facebook (Facebook Authentication Gem)
+gem 'omniauth-facebook'
+#Use certified gem to avoid problems with the ssl cert on localhost
+gem 'certified'
```
Buena. Esta no me la sabía 👍

escoberik replied:
Pero ahora que lo pienso, también va en `group: :development`

jizavala replied:
Agregada gem certified al group development.

escoberik commented on Gemfile:
```diff
@@ -35,6 +41,12 @@ gem 'jbuilder', '~> 2.5'
+
+#Use thor downgrading
+gem 'thor', '0.19.1'
+
```
Y esto para qué es? o.O

jizavala replied:
Se removio esta gema ya que tiene un uso similar al rake.

escoberik reviewed (changes requested):
El código se ve bien en general. Pequeñas correcciones. En cuanto a los assets JS, usamos otra herramienta para manejarlos. Por el momento está bien como lo estás haciendo. En un futuro cercano migramos a la herramienta. Se llama `bower`,  por si tienes tiempo de darle una checada

jizavala committed:
Facebook Login - Pull request.

Se agregaron las gemas wdm y certified al group development.
Se utiliza la gem bybug fuera de los groups development y test ya que al
momento de realizar el debug de la aplicacion se marcaba un error de no
reconocer la gema.
Se removio la gema Thor del gemfile ya que esta tiene una funcionalidad
parecida al Rake.

jizavala committed:
Actualizacion de Perritos

Se agrega el controlador de dogs para el guardado de tag, images y su
avatar correspondiente.
Se agregan los models
Se agregan los migrations

jizavala committed:
Maquetado y visualizacion de Perritos

Se agrego simple_form and client_side_validations Gems to the gemfile.
Se modifico el controller de dog para poder realizar el insert y update
del dog cuando se guarda por primera vez.
Se modifico el initializer de simple_form para que agregara las clases
de bootstrap.
Agrego la vista para mostrar la informacion del dog.
Se modifico el controller de pages para mostrar la info de los
diferentes dogs que se encuentran registrados

jizavala committed:
Cambios del 1 al 15 de Mayo 2017

Se agregaron las gemas jquery upload file y pg, para el guardado
multiple de imagenes y el manejo de postgreSQL respectivamente.
Se agrego el migration file para agregar las columnas de latitud,
longitud y descripcion.
Se modifico el controlador de dogs para la inclusion de las tres nuevas
columnas.
Se agrego una vista parcial que contiene el codigo muestra el menu con
las tags Busco a mi dueño y Busco a mi perro.
Se agregan las views modificadas de pages y dogs para mostrar un layout
de dos columnas cuando se visualiza desde un celular.
Se modifico la view de la accion show, se sube pero aun se encuentra en
desarrollo.
Se modifico la view para las photos, aun esta en pruebas.

jizavala committed:
Multiupload of photos, no zoom in with mobile devices.

Se agrego la gema dropzone a la app para subir de forma masiva imagenes.
Se agrego la gema lightbox para mostrar las imagenes en las galerias de
los perritos registrados.
Se modifico la meta tag del viewport para deshabilitar el zoom in en los
dispositivos moviles.
Se elimino el boostrap que se utilizo en un principio descargandolo del
sitio del mismo.
Se agrego el archivo de bower para la el correcto manejo de las
librerias front end.
Se removio el fileupload ya que presentaba problemas de integracion con
rails.
